### PR TITLE
misc: add xschem

### LINKED ISSUE #249: add xschem package
- Upstream: https://github.com/StefanSchippers/xschem
- License: [GPLv2+](https://github.com/StefanSchippers/xschem/blob/master/LICENSE)
- Recipe: https://xschem.sourceforge.io/stefan/xschem_man/install_xschem.html (standard autotools)
- Deps: x11, cairo, tcl/tk, xpm
- Build-deps: bison, flex

### PR DESCRIPTION
**Pull request by proppy**
Fixes #249

### CODE REVIEW

**Comment by proppy** [comment]
some warnings in the build [logs](https://github.com/hdl/conda-eda/actions/runs/3748964222/jobs/6366903540):
```
2022-12-21T12:22:02.1074422Z Warning: rpath /home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_build_env/lib is outside prefix /home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p (removing it)
2022-12-21T12:22:02.1075781Z Warning: rpath /home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_build_env/lib is outside prefix /home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p (removing it)
2022-12-21T12:22:02.1077219Z Warning: rpath /home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_build_env/lib is outside prefix /home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p (removing it)
2022-12-21T12:22:02.1078528Z Warning: rpath /home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_build_env/lib is outside prefix /home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p (removing it)
2022-12-21T12:22:02.1079811Z Warning: rpath /home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_build_env/lib is outside prefix /home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p (removing it)
```
`_build_env/lib is outside prefix` seems to be expected for `cdt` dependencies since they are in `build` not in `host`, does that mean that we should move them?

```
2022-12-21T12:22:02.1081004Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/libpcrecpp.a)
2022-12-21T12:22:02.1082123Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/libpcre.a)
2022-12-21T12:22:02.1083243Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/libxcb-composite.a)
2022-12-21T12:22:02.1084341Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/libicuuc.a)
2022-12-21T12:22:02.1085455Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/libxcb-xfixes.a)
2022-12-21T12:22:02.1086557Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/libpixman-1.a)
2022-12-21T12:22:02.1087645Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/libpng16.a)
2022-12-21T12:22:02.1088735Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/libxcb-xinput.a)
2022-12-21T12:22:02.1089924Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/libxcb.a)
2022-12-21T12:22:02.1090995Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/libxcb-randr.a)
2022-12-21T12:22:02.1092085Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/libxcb-dri3.a)
2022-12-21T12:22:02.1093170Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/libicutest.a)
2022-12-21T12:22:02.1094266Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/libfontconfig.a)
2022-12-21T12:22:02.1095359Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/libicutu.a)
2022-12-21T12:22:02.1096451Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/libxcb-xvmc.a)
2022-12-21T12:22:02.1097611Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/libxcb-xv.a)
2022-12-21T12:22:02.1098692Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/libz.a)
2022-12-21T12:22:02.1099789Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/libxcb-render.a)
2022-12-21T12:22:02.1100927Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/libcairo-script-interpreter.a)
2022-12-21T12:22:02.1102120Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/libcairo.a)
2022-12-21T12:22:02.1107752Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/libxcb-xf86dri.a)
2022-12-21T12:22:02.1108846Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/libffi.a)
2022-12-21T12:22:02.1109935Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/libxcb-shape.a)
2022-12-21T12:22:02.1111024Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/libpcreposix.a)
2022-12-21T12:22:02.1112100Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/libxcb-xkb.a)
2022-12-21T12:22:02.1113196Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/libxcb-sync.a)
2022-12-21T12:22:02.1114284Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/libtclstub8.6.a)
2022-12-21T12:22:02.1115370Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/libxcb-xtest.a)
2022-12-21T12:22:02.1116532Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/libtkstub8.6.a)
2022-12-21T12:22:02.1117635Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/libxcb-damage.a)
2022-12-21T12:22:02.1118731Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/libxcb-xinerama.a)
2022-12-21T12:22:02.1119830Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/libxcb-glx.a)
2022-12-21T12:22:02.1120906Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/libuuid.a)
2022-12-21T12:22:02.1121961Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/libXpm.a)
2022-12-21T12:22:02.1123040Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/liblzma.a)
2022-12-21T12:22:02.1124125Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/libxcb-dpms.a)
2022-12-21T12:22:02.1125216Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/libxcb-res.a)
2022-12-21T12:22:02.1126326Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/libxcb-screensaver.a)
2022-12-21T12:22:02.1127427Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/libicui18n.a)
2022-12-21T12:22:02.1128499Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/libxcb-shm.a)
2022-12-21T12:22:02.1129655Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/libicuio.a)
2022-12-21T12:22:02.1130752Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/libcairo-gobject.a)
2022-12-21T12:22:02.1131837Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/libicudata.a)
2022-12-21T12:22:02.1132921Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/libfreetype.a)
2022-12-21T12:22:02.1134015Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/libxcb-record.a)
2022-12-21T12:22:02.1135127Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/libxcb-dri2.a)
2022-12-21T12:22:02.1140985Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/libxcb-present.a)
2022-12-21T12:22:02.1142103Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/tdbc1.1.3/libtdbcstub1.1.3.a)
2022-12-21T12:22:02.1143329Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/cairo/libcairo-trace.a)
2022-12-21T12:22:02.1144466Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/itcl4.2.2/libitclstub4.2.2.a)
2022-12-21T12:22:02.1145298Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_build_env/lib/libmenu.a)
2022-12-21T12:22:02.1145936Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_build_env/lib/liby.a)
2022-12-21T12:22:02.1146566Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_build_env/lib/libhistory.a)
2022-12-21T12:22:02.1147212Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_build_env/lib/libformw.a)
2022-12-21T12:22:02.1147850Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_build_env/lib/libopcodes.a)
2022-12-21T12:22:02.1148500Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_build_env/lib/libreadline.a)
2022-12-21T12:22:02.1149142Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_build_env/lib/libncursesw.a)
2022-12-21T12:22:02.1149772Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_build_env/lib/libpanelw.a)
2022-12-21T12:22:02.1150409Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_build_env/lib/libtinfo.a)
2022-12-21T12:22:02.1151053Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_build_env/lib/libctf-nobfd.a)
2022-12-21T12:22:02.1151681Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_build_env/lib/libfl.a)
2022-12-21T12:22:02.1152303Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_build_env/lib/libctf.a)
2022-12-21T12:22:02.1152930Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_build_env/lib/libbfd.a)
2022-12-21T12:22:02.1153639Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_build_env/lib/gcc/x86_64-conda-linux-gnu/11.2.0/libiberty.a)
2022-12-21T12:22:02.1154364Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_build_env/lib/gcc/x86_64-conda-linux-gnu/11.2.0/libgcov.a)
2022-12-21T12:22:02.1155085Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_build_env/lib/gcc/x86_64-conda-linux-gnu/11.2.0/libgcc.a)
2022-12-21T12:22:02.1155804Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_build_env/lib/gcc/x86_64-conda-linux-gnu/11.2.0/libgcc_eh.a)
2022-12-21T12:22:02.1156583Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_build_env/lib/5.34.0/x86_64-linux-thread-multi/CORE/libperl.a)
2022-12-21T12:22:02.1157367Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_build_env/x86_64-conda-linux-gnu/sysroot/usr/lib64/librpcsvc.a)
2022-12-21T12:22:02.1158125Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_build_env/x86_64-conda-linux-gnu/sysroot/usr/lib64/libbsd-compat.a)
2022-12-21T12:22:02.1158894Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_build_env/x86_64-conda-linux-gnu/sysroot/usr/lib64/libpthread_nonshared.a)
2022-12-21T12:22:02.1159643Z WARNING :: Failed to get_static_lib_exports(/home/runner/work/conda-eda/conda-eda/workdir/conda-env/conda-bld/xschem_1671625240647/_build_env/x86_64-conda-linux-gnu/sysroot/usr/lib64/libc_nonshared.a)
```
`Unknown format` error seems to be ok per https://github.com/hdl/conda-eda/pull/165#issuecomment-1059282395.

@ajelinski do you agree with that analysis? does the package looks good enough to merge?

**Comment by mithro** [comment]
Any idea why the CI is failing?

File: misc/xschem/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+requirements:
+  build:
+    - autoconf
+    - automake
+    - make
+    - bison
+    - flex
+    - {{ compiler('c') }}
+    - {{ cdt('xorg-x11-proto-devel') }}  # [linux]
```
Comment (mithro) [inline misc/xschem/meta.yaml:35]: These `cdt` dependencies are required to run the program, right?

I'm pretty sure that `cdt` dependencies are not auto-detected / checked and hence have to be manually specified?

**Comment by proppy** [comment]
> Any idea why the CI is failing?

I suspect it's flaky, but note that the CI is not failing for the `xschem` package being added here: https://github.com/hdl/conda-eda/actions/runs/3748964222/jobs/6366903540

File: misc/xschem/meta.yaml (hunk 1 above)
Comment (proppy) [reply misc/xschem/meta.yaml:35]: yep, my understanding is that those don't need to be added to `run` since they are part of the sysroot.

Comment (proppy) [reply misc/xschem/meta.yaml:35]: and it's similar to what's being done for the magic, netgen and openroad packages:
https://github.com/hdl/conda-eda/blob/master/misc/netgen/meta.yaml#L24-L27
https://github.com/hdl/conda-eda/blob/master/misc/magic/meta.yaml#L29-L33

So if we want to start doing something different from now on, I'd rather unify all of those consistently.

Comment (mithro) [reply misc/xschem/meta.yaml:35]: Yeap, make sense to me.

**Comment by mithro** [comment]
Merge away?

**Comment by proppy** [comment]
I suspect @ajelinski will want us to squash commits first!

**Comment by proppy** [comment]
Rebased and squashed, PTAL

**Comment by proppy** [comment]
> Merge away?

Merged.

**Pull request merged**